src/common/transformations/src/transformations/op_conversions/convert_gather_to_compressed.cpp

```diff
@@ -146,3 +147,46 @@ ov::pass::ConvertGatherToGatherCompressed::ConvertGatherToGatherCompressed() {
     auto m = std::make_shared<ov::pass::pattern::Matcher>(gather_m, "ConvertGatherToGatherCompressed");
     this->register_matcher(m, callback);
 }
+
+ov::pass::MoveDecompressionAfterGather::MoveDecompressionAfterGather() {
+    using namespace ov::pass::pattern;
+
+    auto compressed_constant_without_scale = [](const ov::Output<ov::Node>& output) {
+        return output.get_element_type() == ov::element::f16 || output.get_element_type() == ov::element::bf16;
+    };
+    auto dicts_wo_scale = wrap_type<ov::op::v0::Constant>(compressed_constant_without_scale);
+    auto convert_wo_scale = wrap_type<ov::op::v0::Convert>({dicts_wo_scale}, pattern::consumers_count(1));
+    auto gather_wo_scale =
```

Comment:
Minor: "wo_scale" postfix is not needed anymore, since we already moved this matcher to a separate transformation

Reply:
applied, thanks Vladislav!